### PR TITLE
disabled  module configurations  of turris-omnia

### PR DESCRIPTION
**Pull request by Dhanunjaya-rama**
adding  component dependencies entries at packagegroup-turris-core

### CODE REVIEW

File: recipes-core/packagegroups/packagegroup-turris-core.bb [hunk 1]
```diff
@@ -46,7 +46,12 @@ RDEPENDS_packagegroup-turris-core = " \
    dt \
    u-boot-fw-utils \
     "
-RDEPENDS_packagegroup-turris-core_remove = "dt"
+RDEPENDS_packagegroup-turris-core_remove = "\
+ dt \
+ e2fsprogs \
+ pptp-linux \
```
Comment (manigandanj) [inline recipes-core/packagegroups/packagegroup-turris-core.bb:52]: please remove this line.

File: recipes-core/packagegroups/packagegroup-turris-core.bb [hunk 2]
```diff
@@ -46,7 +46,12 @@ RDEPENDS_packagegroup-turris-core = " \
...
+ rp-pppoe \
```
Comment (manigandanj) [inline recipes-core/packagegroups/packagegroup-turris-core.bb:53]: please remove this line.

File: recipes-core/packagegroups/packagegroup-turris-core.bb [hunk 3]
```diff
@@ -46,7 +46,12 @@ RDEPENDS_packagegroup-turris-core = " \
    dt \
    u-boot-fw-utils \
     "
-RDEPENDS_packagegroup-turris-core_remove = "dt"
+RDEPENDS_packagegroup-turris-core_remove = "\
+ dt \
+ e2fsprogs \
```
Comment (manigandanj) [inline recipes-core/packagegroups/packagegroup-turris-core.bb:51]: please remove this line.

**Comment by Dhanunjaya-rama** [comment]
this changes  has merged by another pull request , it is not merging further . so closing thus this pull request

**Comment by Dhanunjaya-rama** [comment]
closind this pull request

**Pull request closed without merging**